Packaging.props

```diff
@@ -20,7 +20,7 @@
     <PlatformPackageVersion Condition="'$(PlatformPackageVersion)' == ''">1.2.0</PlatformPackageVersion>
 
     <!-- by default all packages will use the same version which revs with respect to product version -->
-    <PackageVersion Condition="'$(PackageVersion)' == ''">4.4.0</PackageVersion>
+    <PackageVersion Condition="'$(PackageVersion)' == ''">4.4.1</PackageVersion>
```

Comment:
Do you really want to update all packages? I would suggest just updating the one you want to ship and not build the others.

Reply:
That is what we do in corefx. We only add the packages we want to ship to the packages.builds file https://github.com/dotnet/corefx/blob/release/2.0.0/src/packages.builds#L19.

Reply:
I think we should respin all WCF packages because people may not use the S.SM.Primitives package directly.

Reply:
Of course that is your choice.

Reply:
After discussed with @weshaggard offline, I think we can go with the plan to rebuild System.ServiceModel.Primitives only as the compat pack will be updated to reference the newest version of this package. The compat pack is what customers will use in the end.